pkgs/desktops/gnome/extensions/dash-to-panel/default.nix

```diff
@@ -1,31 +0,0 @@
-{ lib, stdenv, fetchFromGitHub, glib, gettext }:
-
-stdenv.mkDerivation rec {
-  pname = "gnome-shell-extension-dash-to-panel";
-  version = "45";
-
-  src = fetchFromGitHub {
-    owner = "home-sweet-gnome";
-    repo = "dash-to-panel";
-    rev = "v${version}";
-    sha256 = "05bfd3b1g9zd86pl1rpgfqsmip271lasyfj8phpqf1gdds5yz6f6";
-  };
-
-  buildInputs = [
-    glib gettext
-  ];
-
-  makeFlags = [ "INSTALLBASE=$(out)/share/gnome-shell/extensions" ];
```

Comment:
Looking at the `Makefile`, you could just pass `"VERSION=${version}"` to `makeFlags`:

https://github.com/home-sweet-gnome/dash-to-panel/blob/940215555b67bd79d9e87f79b4961bd94cf31a45/Makefile#L94-L95

Reply:
Thank you for the info. I mistaken the gnome extension name from your PR #160726 🤦🏻‍♂️. Getting it directly from <https://extensions.gnome.org> fixes that issue.